docs/make.bat

```diff
@@ -0,0 +1,263 @@
+@ECHO OFF

```

Comment:
A kind of makefile yes, but for windows. The docs directory is what you get by using `sphinx-quickstart`. It provides all the files needed to generate the sphinx documentation.